maintainers/upload-release.pl

```diff
@@ -117,7 +117,16 @@ sub downloadFile {
     my $dstKey = "$releaseDir/" . $name;
     unless (defined $releasesBucket->head_key($dstKey)) {
         print STDERR "uploading $fn to s3://$releasesBucketName/$dstKey...\n";
-        $releasesBucket->add_key_filename($dstKey, $fn)
+
+        my $configuration = ();
```

Comment:
```suggestion
        my $configuration = {};
```

(I think, I didn't test.)

Reply:
I borrowed this from https://github.com/NixOS/nixos-channel-scripts/blob/master/mirror-nixos-branch.pl#L277